packages/design-tokens/package.json

```diff
@@ -17,7 +17,9 @@
     "js",
     "dist"
   ],
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.cjs",
+  "module": "dist/esm/index.mjs",
+  "types": "dist/types/index.d.ts",
   "private": false,
   "license": "MIT",
   "sideEffects": false,
```

Comment:
Might need to review the sideEffects of this package if this now supports tree-shaking

Reply:
All JS tokens are in a single file anyways, nothing to treeshake... yet...